docs/reference/changelog.rst

```diff
@@ -34,6 +34,12 @@ Bug fixes:
   clean shared areas (stage and prime directories) in partitions. Also make sure
   to account for content coming from the overlay.
 
+.. note::
+
+    2.14.0 includes changes from the 2.10.1 release.
```

Comment:
_Must_ it be a note? :)

Reply:
Certainly not, I was following precedent in this file. 

Would you prefer regular text?